images/toolbox/Dockerfile.base

```diff
@@ -31,4 +31,13 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get upgrade -y && \
```

Comment:
If we want to shrink a little we could remove `vim` and use `vim-tiny`, also why are `fio`, `sudo`, `s3cmd`, `iperf3`, `gdb` needed?

Reply:
@leseb The toolbox image contains tools to run benchmarks like `fio` and use the object store `s3cmd`. `gdb` for debugging.

Reply:
@galexrt, I understand their purpose I'm just questioning whether or not they should be present in this particular image. I know debug tools can be handy to have but they will take unnecessary space when things are running well. Also, they could be installed for debug purpose only with `apt-get`.

Reply:
I think the intention is that the toolbox is used for diagnostics/debugging, and I would think that an editor and some core tools would be used frequently. Image size is less of a concern with the toolbox.

Reply:
Oops my bad, didn't notice it was a different image. Sorry for the noise.